pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -313,7 +313,7 @@ func (p *timeSeriesQueryParser) parse(tsdbQuery *tsdb.TsdbQuery) ([]*Query, erro
 			return nil, err
 		}
 		alias := model.Get("alias").MustString("")
-		interval := strconv.FormatInt(q.IntervalMs, 10) + "ms"
+		interval := model.Get("interval").MustString("0ms")
```

Comment:
Should be empty per default for the fallback to configured data source min interval or system default works as expected.

```suggestion
		interval := model.Get("interval").MustString("")
```